Processor/Db/JobDb.sql

```diff
@@ -0,0 +1,174 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 15.2
+-- Dumped by pg_dump version 15.2
+
+-- Started on 2023-03-07 07:43:08 EET
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+SET default_tablespace = '';
+
+SET default_table_access_method = heap;
+
+--
+-- TOC entry 216 (class 1259 OID 16437)
+-- Name: Job; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."Job" (
+    "Id" uuid NOT NULL,
+    "JobTypeId" integer NOT NULL,
+    "Created" timestamp without time zone NOT NULL,
+    "CreatorId" integer,
+    "JobStatusId" integer NOT NULL,
+    "Parameters" text,
+    "TenantId" uuid NOT NULL DEFAULT '00000000-0000-0000-0000-000000000000'::uuid
+);
+
+
+ALTER TABLE public."Job" OWNER TO fiber_pim_dev;
```

Comment:
Please update all names to something more general

Reply:
done